src/DocsOverview.res

```diff
@@ -30,35 +30,8 @@ let default = (~showVersionSelect=true) => {
     ("Reanalyze", "https://github.com/rescript-lang/reanalyze"),
   ]
 
-  let versionSelect = if showVersionSelect {
-    let onChange = evt => {
-      open Url
-      ReactEvent.Form.preventDefault(evt)
-      let version = (evt->ReactEvent.Form.target)["value"]
-      let url = Url.parse(router.route)
-
-      let targetUrl =
-        "/" ++
-        (Array.join(url.base, "/") ++
-        ("/" ++ (version ++ ("/" ++ Array.join(url.pagepath, "/")))))
-      router->Next.Router.push(targetUrl)
-    }
-
-    <div className="text-fire">
-      <VersionSelect
-        availableVersions=Constants.allManualVersions
-        nextVersion=?Constants.nextVersion
-        onChange
-        version
-      />
-    </div>
-  } else {
-    React.null
-  }
-
```

Comment:
I didn't know this page. Removing version select because it does not seem to be necessary

https://rescript-lang.org/docs/v11.0.0